docs/sales-billing/student.md

```diff
@@ -4,4 +4,13 @@ title: Student Discount
 sidebar_label: Student Discount
 ---
 
-## TODO - Student Discount
+![Subscribing](https://i.imgur.com/JWmOSir.png)
+
+## Getting the discount
+20% student discount is guaranteed to accounts created using a student e-mail address.
+Tryhackme supports all student e-mail addresses and automatically recognizes domains like **.edu** and **.ac.uk**
```

Comment:
Can you make this `StudlyCaps`, i.e. TryHackMe instead of Tryhackme.

Reply:
Fixed that.

Reply:
LGTM :tada: